_includes/releases/left_sidebar.html

```diff
@@ -15,6 +15,9 @@
 -->
 {% include left_sidebar_toggle.html %}
 <div id="left-sidebar-links" class="collapse">
+    <div class="left-sidebar-group">
```

Comment:
I think we should do something like:

```
Release Management
     Versioning
     Artifacts
```

This section would go below v0.4, not above, since v0.4 is more important and contains the default page.

Reply:
Updated the sidebar and split into two files